content/2016/09/30/cssconf.rst

```diff
@@ -0,0 +1,640 @@
+Thanks to `Nicole Sullivan`_ for founding the series,
+`Bocoup`_ for hosting the event,
+(OddBird alum) `Claudina Sarahe`_ for her amazing emcee-ing,
+all the other speakers for their inspiring talks,
+and everyone else for attending!
+It was a great atmosphere,
+and I met all sorts of lovely people.
+
+My talk was called *Sass Map Magic* —
+a major update to a talk I had given
+several years back
+at BlendConf (may it rest in peace).
+The `slides`_ are all online,
+and video will be available soon.
+
+I took extensive notes on day one,
+but lost my focus on day two
+since I was part of the lineup.
+All the talks were great,
+and worth your time
+if you get a chance to watch the videos.
+
+.. _Nicole Sullivan: https://twitter.com/stubbornella
+.. _Bocoup: https://twitter.com/bocoup
+.. _Claudina Sarahe: http://twitter.com/itsmisscs
+.. _slides: https://oddbooksapp.com/book/sass-map-magic
+
+
+Creativity in Programming for Fun and Profit
+--------------------------------------------
+
+.. image:: /static/images/blog/cssconf-2016/sd.jpg
+
+`Sarah Drasner`_
+talked about programming as a creative act.
+There is more than one way to solve any given problem —
+and different solutions will come with different trade-offs.
+It's worth exploring the options,
+while keeping in mind that code is a communication tool,
+and you should engineer with maintainers in mind.
+
+In the second part of the talk,
+Sarah listed several tools for creative thinking.
+I have found all these tools particularly helpful as an artist,
+and it was a nice reminder that they can apply to code as well.
+
+1. Question the base premises
+2. Impose artificial bounds
+3. Repurpose existing solutions
+4. Combine existing solutions
+5. Use open-source code (this seems like an extension of 3 and 4)
+
+Of course,
+this summary barely does justice
+to the beautiful demonstrantions Sarah included.
+Check out her `CodePen`_ page
+for a whole load of amazing and beuatiful samples.
+
+.. _Sarah Drasner: http://twitter.com/sarah_edo
+.. _CodePen: http://codepen.io/sdras/
+
+
+No Bugs in Sight
+----------------
+
+.. image:: /static/images/blog/cssconf-2016/bj.jpg
+
+`Brian Jordan`_
+talked about automated front-end testing,
+with tools like PhantomJS.
+He works with `code.org`_,
+a non-profit working to increase student access
+to computer science curriculum
+across demographics.
+
+I didn't get good notes on his overview of PhantomJS,
+but there were a few philisophical points that stood out:
+
+- Test everything
+- Make sure everyone on the team can write tests
+  for the code they write
+- Take the time, on occasion,
+  to review and clean up your tests —
+  otherwise you will run into bloat and compile-time issues
+
+.. _Brian Jordan: http://twitter.com/bcjordan
+.. _`code.org`: http://code.org
+
+
+Nativize Is the New Normalize
+-----------------------------
+
+.. image:: /static/images/blog/cssconf-2016/jl.jpg
+
+`Jessica Lord`_
+gave one of my favorite talks
+that I didn't understand.
```

Comment:
Was it your favorite talk? Or just your favorite out of the ones you didn't understand?